### ISSUE TITLE
use properties

### ISSUE THREAD

**Issue by arsenovic** [brombo/galgebra#23]
allan, would you consider making all the methods of Mv that dont take any arguments `properties`?

**Comment by moble** [comment]
It might help to point out what this means, by way of example.  In the `MV` class, there is a method that starts with
```python
    def base_rep(self):
```
and gets called as `mv.base_rep()`.  Note the parentheses at the end.  If you change the method to start with
```python
    @property
    def base_rep(self):
```
you will now be able to get the exact same result with `mv.base_rep`.  Note the lack of parentheses.

This is handy, can make code look nicer, and frequently makes sense.  The main drawback is that old code that still has `mv.base_rep()` in it now has extra parentheses.  In this case, `base_rep` returns a `MV` object, so that parentheses calls the object as a function (with no arguments).  This will either not give you what you want, or will just error out.

**Comment by arsenovic** [comment]
yes. great  explanation. as a design rule, if i know an attribute will never need an argument i always make it a property.
